e2e/.gitignore

```diff
@@ -0,0 +1 @@
+TestReport*.xml
```

Comment:
This should not even be applicable to local environment, since by default bats is called without the junit formatter. Should this be removed? It doesn't hurt, and maybe someone runs bats with local junit formatter, probably there are some IDE support for junit reports files.

Reply:
I'd keep it.